### DISCUSSION TITLE
[AJUDA] Adicionar Marca, Modelo , NS,

### DISCUSSION THREAD

**Discussion by laertediniz** [RamonSilva20/mapos#2569] [Q&A]
Alguem pode me ajudar na Versão: 4.47.0?
Já fiz as alterações no código, na criei as tabelas no tb, não não imputa dos dados

OBS, na versão MapsOS - 4.44.0 funcinou direitinho,

![image](https://github.com/user-attachments/assets/23c63f47-426a-4637-a27c-008e3080cd78)

**Comment by GiovaniRio** [comment]
Conseguiu resolver este problema? Também estou precisando fazer estas alterações .

**Reply by maycon1977** [reply]
Oi, amigo. Temos um grupo onde alguns desenvolvedores estão pegando esse e outros trabalhos sob demanda...

Me chama no WhatsApp e te passo mais informações.

+5521988078360
